junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvFileSource.java

```diff
@@ -146,4 +146,15 @@
 	@API(status = EXPERIMENTAL, since = "5.7")
 	int maxCharsPerColumn() default 4096;
 
+	/**
+	 * Identifies whether leading and trailing whitespace characters of unquoted CSV columns should be ignored.
```

Comment:
Please hard-wrap this line at 80 characters as per [our conventions](https://github.com/junit-team/junit5/blob/main/CONTRIBUTING.md#javadoc) (same for the other annotation).